SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -285,6 +285,8 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 	FShutdownEvent OnShutdown;
 
+	class USpatialGameInstance* GetGameInstance() const;
```

Comment:
Hmm, so I committed this by accident. But it's useful for debugging as changing spatialnetdriver.h requires 135 actions rebuilt, and making it public lets you do NetDriver->GetGameInstance()->SpatialWorkerId with your logs, which is very handy.

Thoughts on permanently leaving it in public @mironec?

Reply:
I think `Actor->GetGameInstance()` is usually sufficient